src/main/java/controller/LottoController.java

```diff
@@ -21,16 +22,18 @@ public void play() {
 		displayLottoGameResult(lottoGameResult);
 	}
 
-	private static void displayLottoGameResult(LottoGameResult lottoGameResult) {
-		LottoView.displayResult(lottoGameResult);
-		LottoView.displayYield(lottoGameResult.calculateYield());
-	}
+	private static LottoTickets buyLottoTickets() {
+		Cash cash = LottoView.getCash();
+		int countOfManualTickets = LottoView.getCountOfManualTickets();
+		cash.purchaseLottoTickets(countOfManualTickets);
+		LottoTickets lottoTickets = LottoView.getManualLottoTickets(countOfManualTickets);
 
-	private static LottoGameResult runLottoGame(LottoTickets lottoTickets,
-		WinningLottoTicket lottoWinningNumbers) {
-		LottoGame lottoGame = new LottoGame(lottoTickets, lottoWinningNumbers);
-		LottoGameResult lottoGameResult = lottoGame.calculateResult();
-		return lottoGameResult;
+		int countOfRandomTickets = cash.getTicketCount();
+		LottoTickets totalLottoTickets = lottoTickets.plus(LottoTicketGenerator.generateRandomLottoTickets(countOfRandomTickets));
```

Comment:
사소하지만 `manualLottoTickets`, `autoLottoTickets`으로 변수명을 두고 manualLottoTickets.plus(auttoLottoTickets) 와 같이 코드를 표현해주면   로또 티켓들은  수동로또 티켓들과 자동로또 티켓들이 같이 있구나? 라고 이해하기 쉬울 것 같아요. 😃

```suggestion
                 LottoTickets autoLottoTickets = LottoTicketGenerator.generateRandomLottoTickets(countOfRandomTickets);
		LottoTickets totalLottoTickets = manualLottoTickets.plus(autoLottoTickets);
```

Reply:
반영했습니다. https://github.com/next-step/java-lotto-kakao/pull/35/commits/de7632de4e72c03af5ab1948a343c3470981e773